lib/scrabble.rb

```diff
@@ -15,4 +14,28 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def calculate_points(word)
+    uppercase = word.upcase
+    keys = uppercase.split('')
+    keys.map {|key| point_values[key]}
+  end
```

Comment:
I would move this method below score and score_with_multiplier since those are the higher level methods that make up the user interface and calculate_points is a supporting method.

Reply:
Using the variable keys is a bit akin to using the data type. I see how it shows that you plan to use this collection as keys to the hash; however, it doesn't let the reader know the content. letters might be a more descriptive variable name for the array and letter for the block parameter with the .map block.